borgmatic/config/schema.yaml

```diff
@@ -1602,6 +1602,88 @@ properties:
                 example:
                     - start
                     - finish
+    zabbix:
+        type: object
+        additionalProperties: false
+        properties:
+            itemid:
+                type: integer
+                description: |
+                    The itemid to publish to
+                    for details.
+                example: 55105
+            host:
+                type: string
+                description: |
+                    Host name where the item is stored.
+                    Required if 'itemid' is not set
+                example: borg-server
+            key:
+                type: string
+                description: |
+                    Key of the host where the item is stored.
+                    Required if 'itemid' is not set
+                example: borg.status
+            server:
+                type: string
+                description: |
+                    The address of your self-hosted zabbix instance.
```

Comment:
`host` is the Zabbix term used for a server/device that Zabbix is monitoring. A Zabbix `server` might have multiple `hosts` that is monitors.

`server` is in reference to the zabbix server you are sending data to. This should be better: https://github.com/borgmatic-collective/borgmatic/pull/85/commits/1b9f95ca471c1598780a55a2f362d22b4fc9692f

Reply:
Got it. So `host` is like a client, at least from the perspective of borgmatic monitoring? It might be useful to put some of your explanation into the schema (if you haven't already) so the distinction between the two options is clear.